src/constants.ts

```diff
@@ -38,4 +38,5 @@ export enum APP_PATHS {
   WELCOME = '/welcome',
   MY_ACCOUNT = '/myaccount',
   MY_ACCOUNT_EDIT = '/myaccount/edit',
+  PLANTING_SITES = '/planting_sites',
```

Comment:
i think we use '-' instead of '_', see SEEDS_DASHBOARD

Reply:
Oh, you are right!